pkg/operator/csidriveroperator/driverstarter.go

```diff
@@ -112,6 +116,18 @@ func (c *CSIDriverStarterController) sync(ctx context.Context, syncCtx factory.S
 			continue
 		}
 		if !ctrl.running {
+			relatedObjects = append(relatedObjects, configv1.ObjectReference{
+				Group:    operatorapi.GroupName,
+				Resource: "clustercsidrivers",
+				Name:     ctrl.operatorConfig.CSIDriverName,
+			})
+			// We need a special check for Manila to add the OLM namespace
+			if platform == configv1.OpenStackPlatformType {
+				relatedObjects = append(relatedObjects, configv1.ObjectReference{
+					Resource: "namespaces",
+					Name:     "openshift-manila-csi-driver",
+				})
+			}
```

Comment:
Do we still need to carry some code for manila OLM thingy? I think all such operators would have already been migrated. cc @jsafrane

Reply:
This has been removed.